www/pushlink.js

```diff
@@ -44,19 +44,32 @@ var PushLink = function() {
 
 /**
  * Starts PushLink connection
+ * @param {string} packageName - The package name for the app (optional)
  * @param {string} apiKey - Your PushLink API key
  * @param {string} deviceId - The device id
  * @param {function} successCallback - A function to be called if the command succeeded
  * @param {function} errorCallback - A function to be called if the command failed
  */
-PushLink.prototype.start = function(apiKey, deviceId, successCallback, errorCallback) {
+PushLink.prototype.start = function(packageName,apiKey, deviceId, successCallback, errorCallback) {
   'use strict';
 
+  if (!errorCallback){
```

Comment:
I'd suggest the comparison should be something like this:

```js
if (arguments.length === 4 && typeof deviceId === 'function') {
  // then the signature was (apiKey, deviceId, successCallback, errorCallback)
  apiKey = arguments[0];
  deviceId = arguments[1];
  successCallback = arguments[2];
  errorCallback = arguments[3];
}
```

Then we don't need the extra `cordoaExec` call.